config/locales/diaspora/de.yml

```diff
@@ -1245,6 +1245,8 @@ de:
       title: "Stream"
     public:
       title: "Öffentliche Aktivität"
+    local_public:
+      title: "Lokale Posts"
```

Comment:
```suggestion
```

Please don't provide anything but the English translation. Everything else is handled through WebTranslateIt.